components/timestamp/timestamp.tsx

```diff
@@ -90,6 +92,47 @@ function normalizeRangeDescriptor(unit: NonNullable<Props['units']>[number]): Ra
     return unit;
 }
 
+function timezoneDiff(timezone: string | undefined, currentTimezone: string | undefined): ReturnType< typeof FormattedMessage > | undefined {
+    if (!timezone) {
+        return undefined;
+    }
+
+    const currentDate = DateTime.local().setZone(currentTimezone);
+    const userDate = DateTime.local().setZone(timezone);
+
+    const hoursDiff = Duration.fromObject({
+        minutes: userDate.offset - currentDate.offset,
+    }).as('hours');
+
+    if (!hoursDiff) {
+        return undefined;
+    }
+
+    const aheadOrBehind = hoursDiff > 0 ? 'ahead' : 'behind';
+
+    if (aheadOrBehind === 'ahead') {
+        return (
+            <FormattedMessage
+                id='user_profile.account.hoursAhead'
+                defaultMessage=' ({hourDiff} hr. ahead)'
+                values={{
+                    hourDiff: Math.abs(hoursDiff),
+                }}
+            />
+        );
+    }
+
+    return (
+        <FormattedMessage
+            id='user_profile.account.hoursBehind'
+            defaultMessage=' ({hourDiff} hr. behind)'
+            values={{
+                hourDiff: Math.abs(hoursDiff),
+            }}
+        />
+    );
```

Comment:
I went with just decimal hours to make it clear and not take as much space since in theory there should only be 1h and .5h timezone differences. Right? Happy to do whatever @matthewbirtch suggests for that!

On mobile, I'll reply to the other items once I get back!

Reply:
I take back my .5/1h offset comment. Apparently some are 45min which could cause this. 

I still wonder if "1 hr 45 min" is too long compare to "1.75 hr". But your way is definitely more clear to an end user.

Reply:
@calebroseland - I added these suggestions except the separating hours / minutes. See the comment in the main thread.